samples/bluetooth/mesh/light_lc/README.rst

```diff
@@ -0,0 +1,189 @@
+Server by using the Gen. OnOff Server in the third element.
+
+For a more detailed description checkout the documentation for the :ref:`bt_mesh_lightness_srv_readme` and
+:ref:`bt_mesh_light_ctrl_srv_readme`.
+
+The model handling is implemented in :file:`src/model_handler.c`, which uses the :ref:`dk_buttons_and_leds_readme` and the
+:ref:`zephyr:pwm_api` API to control the LEDs on the board.
+
+Requirements
+************
+
+* One of the following development kits:
+
+  * |nRF52DK|
+  * |nRF52840DK|
+
+* Smartphone with Nordic Semiconductor's nRF Mesh mobile app installed in one of the following versions:
+
+  * `nRF Mesh mobile app for Android`_
+  * `nRF Mesh mobile app for iOS`_
+
+User interface
+**************
+
+Buttons:
+   Can be used to input the OOB authentication value during provisioning.
+   All buttons have the same functionality during this procedure.
+
+LEDs:
+   Show the OOB authentication value during provisioning if the "Push button" OOB method is used.
+   First LED outputs the current light level of the Light Lightness server in the second element.
+
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/bluetooth/mesh/light_lc`
+
+.. include:: /includes/build_and_run.txt
+
+.. _bluetooth_mesh_light_testing:
+
+Testing
+=======
+
+After programming the sample to your board, you can test it by using a smartphone with Nordic Semiconductor's nRF Mesh app installed.
+Testing consists of provisioning the device and configuring it for communication with the mesh models.
+
+Provisioning the device
+-----------------------
+
+The provisioning assigns an address range to the device, and adds it to the mesh network.
+Complete the following steps in the nRF Mesh app:
+
+1. Tap :guilabel:`Add node` to start scanning for unprovisioned mesh devices.
+#. Select the :guilabel:`Mesh Light LC` device to connect to it.
+#. Tap :guilabel:`Identify` and then :guilabel:`Provision` to provision the device.
+#. When prompted, select the OOB method and follow the instructions in the app.
+
+Once the provisioning is complete, the app returns to the Network screen.
+
+Configuring models
+------------------
+
+Complete the following steps in the nRF Mesh app to configure models:
+
+1. On the Network screen, tap the :guilabel:`Mesh Light LC` node.
+   Basic information about the mesh node and its configuration is displayed.
+#. In the Mesh node view, expand the third element.
+   It contains the list of models in the third element of the node.
+#. Tap :guilabel:`Generic OnOff Server` to see the model's configuration.
+#. Bind the model to application keys to make it open for communication:
+
+   1. Tap :guilabel:`BIND KEY` at the top of the screen.
+   #. Select :guilabel:`Application Key 1` from the list.
+
+   You are now able to control the first LED on the device by using the Generic On Off Controls in the model view.
```

Comment:
Generic OnOff controls?

Reply:
This is also a knock-off of the documentation in the mesh:light sample. It refers to the interface of the Gen. OnOff Server model interface in the mesh app.
![nrf_mesh_app_gen_onoff_ctrl](https://user-images.githubusercontent.com/35562897/84164989-6fb03000-aa73-11ea-9cc1-656db1e5e8b9.png)